frontend/android/app/build.gradle.kts

```
@@ -6,7 +6,8 @@ plugins {
 }
 
 android {
-    namespace = "com.example.frontend"
+    outputFileName = "BeakleVision-1.0.0.apk"
+    namespace = "org.BeakleVision.frontend"
```

```suggestion
    namespace = "org.beaklevision.frontend"
```

let's probably keep the namespace all lowercase, even though it looks really odd initially